mlblocks/mlpipeline.py

```diff
@@ -73,34 +80,77 @@ def _get_tunable_hyperparameters(self):
 
         return tunable
 
-    def __init__(self, primitives, init_params=None, input_names=None, output_names=None):
-        self.primitives = primitives
-        self.init_params = init_params or dict()
-        self.blocks = OrderedDict()
+    @staticmethod
+    def _build_blocks(primitives, init_params):
```

Comment:
Why make a private static method that is only called by passing instance attributes? Shouldn't it be a regular instance method?